src/Tests/IntegrationTests/ClusterTests.cs

```diff
@@ -126,7 +126,7 @@ public void TestPingResponseError()
                 {
                     // ensure handler on the first conn is called
                     Assert.True(ev.WaitOne(30000));
-                    Assert.Contains("replaced", errStr);
+                    Assert.Contains("Reconnect buffering has been disabled", errStr);
```

Comment:
For this test, we should still check for replaced.  Passing a NATS conn and then placing an AutoResetEvent in a reconnect handler would make the error check deterministic.  The test would wait until the NATS client reconnects (preventing the buffering error) and then should get the error string with "replaced".

Reply:
Fixed